cypress/integration/rich-text/RichTextPage.ts

```diff
@@ -120,6 +120,14 @@ export class RichTextPage {
       .should('be.empty')
       .as('validationErrors');
   }
+
+  expectTrackingvalue(expectedValue: any) {
+    cy.window().then((win) => {
+      setTimeout(() => {
+        expect(win.actions).to.deep.equal(expectedValue);
```

Comment:
This will be a source of flakiness the way it's currently written as Cypress won't re-try assertion. [See this guide](https://docs.cypress.io/guides/core-concepts/retry-ability)

Reply:
This has been improved here: [`07173ea` (#1027)](https://github.com/contentful/field-editors/pull/1027/commits/07173ea2807571c2554a8c5d31014ed4c868b9b0)